socat/astroquery.py

```diff
@@ -43,7 +49,7 @@ async def get_source_info(
         warnings.warn(
             "More than one source resolved, returning first"
         )  # pragma: no cover
-
+    print("KEYS", result_table.keys())
```

Comment:
Remove this, I presume it was for debugging. We can discuss adding logging if that is becoming necessary.